src/Controller/ArticlesController.php

```diff
@@ -1772,10 +1806,10 @@ private function hasPeerReview($item) {
             ->then(function (array $parts) {
                 $item = $parts['item'];
 
-                return ($item instanceof ArticleVoR && $item->getPublicReviews()->notEmpty()) ||
-                    ($item instanceof ArticleVor && $item->getDecisionLetter()) ||
-                    ($item instanceof ArticleVor && $item->getAuthorResponse())
-                ;
+                return $item instanceof ArticleVoR &&
+                        ($item->getPublicReviews()->notEmpty() ||
```

Comment:
Does this need tidying. Tabbed in too many times?

Reply:
It's refactoring, not related to the logic for Process block.